sections/staking/components/StakingInfo/ClearDebtInfo.tsx

```diff
@@ -64,45 +65,41 @@ const StakingInfo: FC = () => {
 			transferableCollateral
 		);
 
-		const changedDebt = debtBalance.isZero() ? zeroBN : debtBalance.minus(amountToBurnBN);
+		const changedDebt = debtBalance.eq(0) ? wei(0) : debtBalance.sub(amountToBurnBN);
 
-		const changedSUSDBalance = sUSDBalance.minus(amountToBurnBN);
+		const changedSUSDBalance = sUSDBalance.sub(amountToBurnBN);
 
-		const changeCRatio = toBigNumber(100).dividedBy(
-			changedDebt.dividedBy(SNXRate).dividedBy(collateral)
-		);
+		const changeCRatio = currentCRatio.neg(); // wei(100).div(changedDebt.div(SNXRate).div(collateral));
```

Comment:
let's remove the comment

Reply:
if you are clearing debt, your C-Ratio goes to 0, right? therefore the change is your negative c ratio

Reply:
ah right of course